src/EnergyPlus/Material.cc

```diff
@@ -2756,6 +2758,99 @@ void GetMaterialData(EnergyPlusData &state, bool &ErrorsFound) // set to true if
     for (auto *m : state.dataMaterial->Material) {
         m->phaseChange = HysteresisPhaseChange::HysteresisPhaseChange::factory(state, m->Name);
     }
+
+    GetMaterialAddOnInput(state, ErrorsFound); // Read variable thermal and solar absorptance add-on data
+}
+
+void ReadingOneMaterialAddOn(EnergyPlusData &state, int i, bool &errorsFound)
```

Comment:
Again, "helper" functions like this don't actually help. They just add overhead because you have to construct and allocate the parameter arrays for each call instead of just once.

Reply:
yeah, previously I was thinking since the thermal and solar one mostly share the same reading procedure, there should be such a helper. Now that they are combined into one idd object, there's no need for this I guess.